pages/index.tsx

```diff
@@ -0,0 +1,18 @@
+import type { NextPage } from 'next';
+import Head from 'next/head';
+import { AppShell, Header } from '@mantine/core';
+import AbacusHeader from '../components/AbacusHeader';
+
+const Home: NextPage = () => {
+  return (
+    <>
+      <Head>
+        <title>FQM Testify: an ECQM Analysis Tool</title>
+      </Head>
+      <AppShell padding="md" header={<Header height={60}>{<AbacusHeader></AbacusHeader>}</Header>}>
+        </AppShell>
```

Comment:
This file gave me errors after I ran prettier. I think AppShell may be required to have children. For now, you could just put an empty `<></> ` React fragment inside